api/dp-openapi.yaml

```diff
@@ -1005,6 +1035,13 @@ components:
         enabled:
           type: boolean
 
+    project_stats:
+      type: object
+      additionalProperties:
+        type: integer
+      description: Project state statistics.
+        Keys are state ID, values are the number of projects in that state
```

Comment:
This description is not true for both, we could probably generalize it and make it work though.